app/components/book-cover.js

```diff
@@ -15,6 +15,18 @@ export default Ember.Component.extend({
     close() {
       this.set('isShowingModal', false);
       this.get('blurBackground')(false);
+    },
+
+    addToCart(title, author, price) {
```

Comment:
The part in this action that takes the title, author & price and adds it to the cart, seems to not be really the scope of this component.
In an idiomatic Ember-world, you'd want to follow DDAU (data-down-action-up) approach and trigger an action to handle the adding to card. 
However, since the cart is a concept that would likely be re-used in various other places....what do you think of having a `cart service` with a clearly defined interface for adding/removing items & clearing the cart?